third_party/packages/split/lib/split.dart

```diff
@@ -125,3 +193,64 @@ StreamSubscription<Object> flexSplitBidirectional(
     createSplitter();
   });
 }
+
+/// Creates a flex splitter that changes from horizontal to vertical depending
+/// on the window aspect ratio.
+///
+/// You should used this fixed splitter instead of the fixed splitter if the
+/// size of the parent element of the element being split isn't fixed. Keep in
+/// mind that the children being split must be sized such that flex-shrink does
+/// not apply as otherwise flex-shrink will interact badly with the calculation
+/// for the size of the two split regions.
+/// https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink
+///
+/// [parts] must be a list of [CoreElement], [Element], or query selectors.
```

Comment:
How would a query selector fit the `Element` list type?

Reply:
Removed all the comments on [parts] as clearly at some point in the past the type of the List was changed but the comment wasn't removed. This is copy and paste from the comment from the other similar method in the file.